src/main/java/nextstep/courses/domain/Image/Image.java

```diff
@@ -0,0 +1,13 @@
+package nextstep.courses.domain.Image;
+
+public class Image {
+    private final ImageCapacity capacity;
+    private final ImageType type;
+    private final ImageSize size;
+
+    public Image(ImageCapacity capacity, ImageType type, ImageSize size) {
```

Comment:
```suggestion
    public Image(int capacity, ImageType type, int width, int height ) {
```
위아 같은 생성자도 추가로 지원하는 것은 어떨까?